### ISSUE TITLE
qcubed supported mysql versions - runs with 5.6.x?

### ISSUE THREAD

**Issue by koopa** [qcubed/qcubed#284]
Has anyone used qcubed with mysql `5.6.x`? 
Are there any caveats to be aware of?
I saw the issue and patch for the new `5.6.x` "EXPLAIN" ( https://github.com/qcubed/framework/pull/157 ), but besides that?

The reason for updating is only that prior to `5.6` mysql _datetime_ types cannot use `CURRENT_TIMESTAMP` as default value, which is needed in our scenario. Or does maybe someone know of an alternative way to achieve the same effect?

Basically we need two automatically updated timestamps for creation and last_updated dates, but two timestamps using `CURRENT_TIMESTAMP` are also not allowed.

Any additional info or hints about this would be great, cheers!
